xml/cl.xml

```diff
@@ -7313,6 +7321,12 @@ server's OpenCL/api-docs repository.
                 <enum name="CL_STRUCTURE_TYPE_MUTABLE_BASE_CONFIG_KHR"/>
                 <enum name="CL_STRUCTURE_TYPE_MUTABLE_DISPATCH_CONFIG_KHR"/>
             </require>
+            <require comment="cl_mutable_dispatch_asserts_khr">
+                <enum name="CL_MUTABLE_DISPATCH_ASSERTS_KHR"/>
+            </require>
```

Comment:
```suggestion
            <require comment="cl_ndrange_kernel_command_properties_khr">
                <enum name="CL_MUTABLE_DISPATCH_ASSERTS_KHR"/>
            </require>
```
`comment` denotes the type, which accepts this enum. I guess this is the parameter:
https://github.com/KhronosGroup/OpenCL-Docs/blob/00bbbcf924d386d62dcc202573abc606659f5653/xml/cl.xml#L3211

Reply:
This enum is reused for both cl_mutable_dispatch_asserts_khr in clCreateCommandBufferKHR and clCommandNDRangeKernelKHR. So, I guess I need both `require`s.

Reply:
Ok, fixed the suggestion.

Reply:
Just FYI, the header generation scripts will generate a block of enums for each "require" section, so if the enum is listed in each block it will be generated twice.  The header generation scripts will handle this (they need to, since sometimes an enum is required by two different extensions) but it does look a little odd if you don't know what's happening:

```c++
/* cl_command_buffer_properties_khr */
#define CL_MUTABLE_DISPATCH_ASSERTS_KHR                     0x12B8

/* cl_ndrange_kernel_command_properties_khr */
/* enum CL_MUTABLE_DISPATCH_ASSERTS_KHR */
```

Note that the second enum is commented out because it is a duplicate.

If you're OK with this, no action required, just thought I'd mention it.  Thanks!